### PR TITLE
fix issue 148 in storm-engine

### PR DESCRIPTION
**Pull request by jjdredd**
https://github.com/storm-devs/storm-engine/issues/148

### CODE REVIEW

File: program/dialog.c [hunk 1]
```diff
@@ -77,7 +77,13 @@ bool DialogMain(ref Character)
 	Dialog.CurrentNode = CharacterRef.Dialog.CurrentNode;
 	startDialogMainCounter = 0;
 	SetEventHandler("frame", "StartDialogMain", 1);
+
 	SetTimeScale(0.0);
+	TimeScaleCounter = -3;
```
Comment (mitrokosta) [inline program/dialog.c]: Should be 0, because time after dialog is reset to 1x

**Review by mitrokosta — CHANGES_REQUESTED** [review]

File: program/dialog.c (hunk 1 above)
Comment (jjdredd) [reply program/dialog.c]: Thanks for reviewing. Should be fixed now.

**Comment by jjdredd** [comment]
@mitrokosta while I'm at it, perhaps I should also patch the same bug on the `develop-rus` branch?
https://github.com/storm-devs/sd-teho-public/blob/develop-rus/program/dialog.c#L80

**Comment by mitrokosta** [comment]
Yeah, fix it in russian branch too please

**Pull request merged**